test/performance/odp_sched_pktio.c

```diff
@@ -534,6 +537,7 @@ static void print_usage(const char *progname)
 	       "  -i, --interface <name>    Packet IO interfaces (comma-separated, no spaces)\n"
 	       "  -c, --num_cpu <number>    Worker thread count. Default: 1\n"
 	       "  -q, --num_queue <number>  Number of pktio queues. Default: Worker thread count\n"
+	       "  -b, --burst <number>      Maximum number of events requested from scheduler. Default: 32\n"
```

Comment:
It's not clear that this patch is related to the others in this PR. It seems to have nothing to do with scheduler groups. This seems a reasonable improvement for this test but shouldn't this be its own PR?

Reply:
Cover letter says: "Sched_pktio burst size change was needed to test the performance effect."

Reply:
OK